docs/ccoctl.md

```diff
@@ -215,3 +215,124 @@ This command will delete the service id from the IBM Cloud
 ```bash
 ccoctl ibmcloud delete-service-id --credentials-requests-dir <path-to-directory-with-list-of-credentials-requests> --name <name> 
 ```
+
+## Alibaba Cloud
+
+This is a guide for using manual mode on alibaba cloud, for more info about manual mode, please refer to [cco-mode-manual](https://github.com/openshift/cloud-credential-operator/blob/master/docs/mode-manual-creds.md).
+
+For alibaba cloud,  the CCO utility (`ccoctl`) binary will create credentials Secret manifests for the OpenShift installer. It will also create a user along with a long-lived RAM AccessKey (AK) for each OpenShift in-cluster component. Every RAM user who owns the AK would be attached RAM policies with the permission defined in each component's CredentialsRequest. To do all this, ccoctl consumes AK of a root RAM user with permissions required for creating user/policy and also attaching the policy to the user.
+
+### Prerequisite
+
+1. Extract and prepare the ccoctl binary from the release image.
```

Comment:
Show a command for how to do this; don't assume you users know how to do this.

Reply:
Agreed, we show this for the AWS versions and I will be modelling the structure for new platform support on what we have for AWS

Reply:
@sferich888 thanks for the review! I could only found a Chinese doc for these on https://help.aliyun.com/document_detail/311677.html, and I will check if there is an English version with my colleague